_components/configuration-component.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Configuration component
+layout: article
+section: Utility Components
+---
+
+## Description
+
+{{site.data.tenant.name}} configuration component which allows separating the
+modification of configurable values from the modification of the flow.
+
+### How works.  API version / SDK version
+
+Configuration component has a credential which is a text field with some valid
+JSON. (JSON must be valid in order for credentials to be verified). It emits a
```

Comment:
JSON must also be less than 5KB in size.

Reply:
thanks @jhorbulyk I will address this now